setup.py

```diff
@@ -51,32 +51,32 @@ def run_tests(self):
 requires = []
 test_requirements = ['pytest>=2.8.0', 'pytest-httpbin==0.0.7', 'pytest-cov', 'pytest-mock']
 
-with open('requests/__init__.py', 'r', 'utf-8') as fd:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
-                        fd.read(), re.MULTILINE).group(1)
-
-if not version:
-    raise RuntimeError('Cannot find version information')
-
 with open('README.rst', 'r', 'utf-8') as f:
     readme = f.read()
 with open('HISTORY.rst', 'r', 'utf-8') as f:
     history = f.read()
 
+metadata = {}
+with io.open('requests/__about__.py', encoding='utf-8') as f:
+    exec(f.read(), metadata)
+
+if not metadata['version']:
+    raise RuntimeError('Cannot find version information')
+
 setup(
     name='requests',
-    version=version,
+    version=metadata['version'],
     description='Python HTTP for Humans.',
     long_description=readme + '\n\n' + history,
-    author='Kenneth Reitz',
+    author=metadata['author'],
     author_email='me@kennethreitz.com',
     url='http://python-requests.org',
     packages=packages,
     package_data={'': ['LICENSE', 'NOTICE'], 'requests': ['*.pem']},
     package_dir={'requests': 'requests'},
     include_package_data=True,
     install_requires=requires,
-    license='Apache 2.0',
+    license=metadata['license'],
```

Comment:
i dont' mind having these hardcoded in, as it makes it easier for people to copy/paste from us... i'll think about it and plan to merge my pr for this